digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/asset/asset.scala

```diff
@@ -978,21 +978,22 @@ case object CyclingAndWalking extends AssetTypeInfo { val typeId = 440; def geom
 case object Lanes extends AssetTypeInfo { val typeId = 450; def geometryType = "linear"; val label = "Lanes" ; val layerName = "lanes"; val nameFI = "Kaista"}
 
 
-object AutoGeneratedValues {
+object AutoGeneratedValues extends Enumeration {
+
+  val dr1Conversion = Value("dr1_conversion")
+  val automaticCorrection = Value("automatic_correction")
+  val excelDataMigration = Value("excel_data_migration")
+  val automaticGeneration = Value("automatic_generation")
+  val generatedInUpdate = Value("generated_in_update")
+  val automaticAdjustment = Value("automatic_adjustment")
+  val mtkClassDefault = Value("mtk_class_default")
+  val autoGeneratedLane = Value("auto_generated_lane")
+  val startDateImporter = Value("start_date_importer")
+  val annualUpdate = Value("annually_updated_period")
```

Comment:
Mikä hyöty tulee että käytetään tätä Value(...) tyyliä? Toimisiko pelkästään val dr1Conversion = "dr1_conversion", niin ei olisi tarvetta käyttää .toString metodia kaikkialla missä näitä käytetään?

Reply:
Varmaan joo. Näin oli jossain scalan enum-tutoriaalissa, mut ehkä toi pelkkä string riittää meidän tapauksessa. Muokkaan tän.